Tests/test_imageshow.py

```diff
@@ -1,57 +1,61 @@
-import unittest
-
+import pytest
 from PIL import Image, ImageShow
 
-from .helper import PillowTestCase, hopper, is_win32, on_ci, on_github_actions
+from .helper import hopper, is_win32, on_ci, on_github_actions
+
+
+def test_sanity():
+    dir(Image)
+    dir(ImageShow)
+
+
+def test_register():
+    # Test registering a viewer that is not a class
+    ImageShow.register("not a class")
+
+    # Restore original state
+    ImageShow._viewers.pop()
 
 
-class TestImageShow(PillowTestCase):
-    def test_sanity(self):
-        dir(Image)
-        dir(ImageShow)
+def test_viewer_show():
+    class TestViewer(ImageShow.Viewer):
+        methodCalled = False
 
-    def test_register(self):
-        # Test registering a viewer that is not a class
-        ImageShow.register("not a class")
+        def show_image(self, image, **options):
+            self.methodCalled = True
+            return True
 
-        # Restore original state
-        ImageShow._viewers.pop()
+    viewer = TestViewer()
+    ImageShow.register(viewer, -1)
 
-    def test_viewer_show(self):
-        class TestViewer(ImageShow.Viewer):
-            methodCalled = False
+    for mode in ("1", "I;16", "LA", "RGB", "RGBA"):
+        with hopper() as im:
+            assert ImageShow.show(im)
+        assert viewer.methodCalled
 
-            def show_image(self, image, **options):
-                self.methodCalled = True
-                return True
+    # Restore original state
+    ImageShow._viewers.pop(0)
 
-        viewer = TestViewer()
-        ImageShow.register(viewer, -1)
 
-        for mode in ("1", "I;16", "LA", "RGB", "RGBA"):
-            with hopper() as im:
-                self.assertTrue(ImageShow.show(im))
-            self.assertTrue(viewer.methodCalled)
+@pytest.mark.skipif(
+    not on_ci() or not (is_win32() and on_github_actions()),
```

Comment:
This was:
```python
    @unittest.skipUnless(
        on_ci() and not (is_win32() and on_github_actions()),
        "Only run on CIs; hangs on Windows on GitHub Actions",
    )
```

So inverting for `skipif`:

```suggestion
    not on_ci() or (is_win32() and on_github_actions()),
```

Reply:
Thanks